### PR TITLE
Add the Lucky command

### PR DESCRIPTION
**Pull request by greenbigfrog**

### CODE REVIEW

File: src/discordtipbot/discordbot.cr [hunk 1]
```diff
@@ -401,10 +407,44 @@ class DiscordBot
...
+
+    users = active_users(msg).to_a
+
+    return reply(msg, "**ERROR**: There is no one to make lucky!")
+
+    user = users.sample
+
+    case @tip.transfer(from: msg.author.id, to: user, amount: amount, memo: "lucky")
+    when "success"
+      return reply(msg, "#{msg.author.username} luckily rained **#{amount} #{@config.coinname_short}** onto **<@#{user}>**")
```
Comment (z64) [inline src/discordtipbot/discordbot.cr]: Each of these `return` is not needed.

File: src/discordtipbot/discordbot.cr [hunk 2]
```diff
@@ -401,10 +407,44 @@ class DiscordBot
     @tip.get_config(guild_id(msg), "mention") || false
   end
 
+  def lucky(msg)
+    return reply(msg, "**ERROR**: This command doesn't work in DMs") if private?(msg)
+
+    cmd_usage = "#{@config.prefix}lucky [amount]"
+
+    # cmd[0]" command, cmd[1]: amount"
```
Comment (z64) [inline src/discordtipbot/discordbot.cr]: Isn't that `"` supposed to be a `:`?

Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: good catch

File: src/discordtipbot/discordbot.cr (hunk 1 above)
Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: btw this isn't the only place I've done this... You missed it in `#soak` and `#rain` and `#tip`

**Pull request merged**